docs/developer/bulk-data-entry-plugin/manual-setup.mdx

```diff
@@ -0,0 +1,108 @@
+
+```json title="capture/bulkDataEntry"
+{
+    "version": 1,
+    "config": [
+        {
+            "title": {
+                "en": "Routine visit"
+            },
+            "dataKey": "routineVisit",
+            "subtitle": {
+                "en": "Covers all patient visit forms"
+            },
+            "configKey": "routineVisit",
+            "programId": "IpHINAT79UW",
+            "pluginSource": "https://debug.dhis2.org/dev"
+        },
+        {
+            "title": {
+                "es": "Tratamiento",
+                "en": "Treatment",
+                "fr": "Traitement",
+                "nb": "Behandling"
+            },
+            "configKey": "treatment",
+            "programId": "IpHINAT79UW",
+            "pluginSource": "https://developers.dhis2.org/docs/capture-plugins/developer/getting-started"
+        },
+        {
+            "title": {
+                "en": "Case based surveillance"
+            },
+            "subtitle": {
+                "en": "Covers ad-hoc visits and RMNCH238972"
+            },
+            "configKey": "caseBasedSurveillance",
+            "programId": "uy2gU8kT1jF",
+            "pluginSource": "https://docs.dhis2.org/en/home.html"
+        }
+    ]
+}
+```
+
+### Configuration Fields
+
+Each configuration object requires the following fields:
+
+| Field | Required | Description |
+|-------|----------|-------------|
+| `configKey` | Yes | An identifier for the configuration. Must be unique per `programId`. Will be passed to the plugin. |  
+| `programId` | Yes |  The identifier of the program where this bulk data entry will be available. |
+| `pluginSource` | Yes | The URL of the plugin. Must point to a valid plugin installed on the instance. |
+| `title` | Yes | The display title of the configuration. Supports multiple languages using locale codes (e.g., "en", "es"). |
+| `subtitle` | No | An optional subtitle for the configuration. Supports multiple languages using locale codes. |
+| `dataKey` | No | An optional identifier that is not used internally by the app but is passed through to the plugin. This property is intended for advanced usage and can be used by the plugin to maintain separate in-progress caches for a given `configKey`. |
+
+## Selecting a Plugin Configuration
+
+Once configured, bulk data entry plugins are accessible in multiple locations throughout the Capture app. You'll find them displayed as widgets on both the Program overview and Search pages:
+
+![Widget bulk data entry main page](../resources/widget-bulk-data-entry-main-page.png)
+![Widget bulk data entry search page](../resources/widget-bulk-data-entry-search-page.png)
+
+The plugins can also be accessed via dropdown menus from the tracked entities working lists or the program stages working lists:
```

Comment:
I wasn't able to get this to work with a plugin I made; is this screenshot below where one would find that option?

<img width="1202" alt="capture-enrollment-view" src="https://github.com/user-attachments/assets/bbe81049-369a-4a9c-9385-3befa9e04134" />